packages/atoms/src/Select.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { FlagIconName, Icon } from '@trezor/icons';
+
+import { Box } from './Box';
+import { Text } from './Text';
+import { BottomModal } from './Modal/BottomModal';
+import { SelectItem, SelectItemProps } from './SelectItem';
+
+export type SelectItem = {
+    label: string;
+    value: string;
+    iconName: FlagIconName;
+};
+
+type SelectProps = {
+    items: SelectItem[];
+    label: string;
+};
+
+const selectStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    backgroundColor: utils.colors.gray200,
+    borderWidth: utils.borders.widths.small,
+    borderRadius: utils.borders.radii.small,
+    borderColor: utils.colors.gray200,
+    color: utils.colors.gray600,
+    paddingLeft: 12,
+    paddingRight: 23.25,
+    height: 58,
+}));
+
+export const Select = ({ items, label }: SelectProps) => {
+    const { applyStyle } = useNativeStyles();
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+    const [selectedItem, setSelectedItem] = useState<null | SelectItemProps['value']>(null);
+
+    const handleSelectItem = (value: string) => {
+        setSelectedItem(value);
+        setIsOpen(false);
+    };
+
+    const getLabel = (): string => {
+        if (!selectedItem) return label;
```

Comment:
is this line even neccessary? I think if you delete it should work same because `?? label` on next line

Reply:
Good point! Missed this one

Reply:
[ad89ef9](https://github.com/trezor/trezor-suite/pull/5506/commits/ad89ef9ca39b59768143099cf46c5230b79c28fb)